routes/index.js

```diff
@@ -1,5 +1,11 @@
+"use strict"
 var express = require('express');
 var router = express.Router();
+var pgp = require('pg-promise')();
+var pq = require('pg-promise').ParameterizedQuery;
+
+const connectionstring = `postgres://${process.env.POSTGRES_USER}:${process.env.POSTGRES_PASSWORD}@${process.env.POSTGRES_HOST}:${process.env.POSTGRES_PORT}/${process.env.POSTGRES_DB}`;
+const db = pgp(connectionstring);
```

Comment:
See the section under _Create_

http://mherman.org/blog/2015/02/12/postgresql-and-nodejs/